packages/dart_frog_cli/lib/src/dev_server_runner/dev_server_runner.dart

```diff
@@ -95,6 +97,10 @@ class DevServerRunner {
   /// Which port number the server should start on.
   final String port;
 
+  /// Which host the server should start on.
+  /// It will default to localhost if empty.
```

Comment:
**Nit:** Start with a single sentence (https://dart.dev/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary)
```suggestion
  /// Which host the server should start on.
  ///
  /// It will default to localhost if empty.
```